plugins/email/src/main/java/de/unistuttgart/iaas/amyassist/amy/plugin/email/EMailLogic.java

```diff
@@ -285,6 +302,35 @@ boolean isImportantMessage(Message message) throws MessagingException {
 		return importantAddresses.contains(address.getAddress());
 	}
 
+	/**
+	 * Set up connection to mail server with given parameters. Put empty strings into all parameters if you want to
+	 * connect with the Amy mail account
+	 * 
+	 * @param credentials
+	 *            the email credentials
+	 * 
+	 * @return true if connecting was successful, else false
+	 */
+	public boolean connectToMailServer(EMailCredentials credentials) {
+		if (credentials.getUsername().isEmpty() && credentials.getPassword().isEmpty()
+				&& credentials.getImapServer().isEmpty()) {
+			// if no credentials are given, use the standard ones for Amy
+			String amyUsername = this.configLoader.getProperty(AMY_MAIL_ADDRESS_KEY);
+			String amyPassword = this.configLoader.getProperty(AMY_MAIL_PW_KEY);
+			String amyHost = AMY_MAIL_HOST;
+
+			return this.mailSession.startNewMailSession(new EMailCredentials(amyUsername, amyPassword, amyHost));
```

Comment:
change this line to `credentials = new EMailCredentials(amyUsername, amyPassword, amyHost);`